websites/P/Perplexity/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+		clientId: "893871116945350726",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	threadExportRegex = /([a-f\d]{8}(-[a-f\d]{4}){3}-[a-f\d]{12})/i;
+let recentSearchQuery: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "lg",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting<boolean>("privacy"),
+		hideButtons = await presence.getSetting<boolean>("hideButtons"),
```

Comment:
Please implement these like this:
https://github.com/PreMiD/Presences/blob/b27818caa4ead98827c810d70911f769e5929cb4/websites/O/OK/presence.ts#L87-L92